lib/core/SimplifiedMasternodeList.js

```diff
@@ -26,10 +28,10 @@ class SimplifiedMasternodeList {
 
   /**
    *
-   * @return {SimplifiedMNList|undefined}
+   * @return {SimplifiedMNList}
```

Comment:
It can be undefined, right after the instance was created

Reply:
Yeah, but we usually do not define it as a type? It's just not present/defined.